### DISCUSSION TITLE
I'm confused about the official state of 64-bit distros.  What's the official stance?

### DISCUSSION THREAD

**Discussion by YamiYukiSenpai** [RaspAP/raspap-webgui#1360] [General]
The reason why I'm asking is because I read in the prerequisites that 64-bit is **not** supported.  However, I've been reading that people have been using 64-bit OS's with this tool.

**Comment by billz** [comment]
Both the 32- and 64-bit RPi OS Lite versions are supported https://docs.raspap.com/#compatible-operating-systems
The project README had not been updated to reflect this (it is now). Thanks for pointing this out

**Comment by CabreraC2** [comment]
I setup my Raspberry pi 4 with bullseye 64-bit as a repeater with Raspap+AdguardHome+Samba 3 weeks ago and still running.
